docs/pages/setup/guides/teleport-operator.mdx

```diff
@@ -0,0 +1,213 @@
+
+At this point the Teleport Kubernetes Operator is functional and Teleport users and roles can be managed from
+Kubernetes.
+
+## Step 3/3. Explore the Teleport CRDs
+
+Available fields can be browsed with `kubectl explain` in a cluster with Teleport CRDs installed.
+For example the command:
+```code
+$ kubectl explain teleportroles.spec
+```
+Returns the following fields:
+```shell
+KIND:     TeleportRole
+VERSION:  resources.teleport.dev/v5
+
+RESOURCE: spec <Object>
+
+DESCRIPTION:
+    Role resource definition v5 from Teleport
+
+FIELDS:
+   allow	<Object>
+     Allow is the set of conditions evaluated to grant access.
+
+   deny	<Object>
+     Deny is the set of conditions evaluated to deny access. Deny takes priority
+     over allow.
+
+   options	<Object>
+     Options is for OpenSSH options like agent forwarding.
+```
+
+## Going Further
+
+### Troubleshooting a resource not reconciled in Teleport
+
+The operator watches for new resources or changes in Kubernetes. When a change happens, it triggers the reconciliation
+loop. This loop is in charge of validating the resource, checking if it already exists in Teleport and making calls to
+the Teleport API to create/update/delete the resource. The reconciliation loop also adds a `status` field on the
+Kubernetes resource.
+
+If an error happens and the reconciliation loop is not sucessful, an item in `status.conditions` will describe what
+went wrong. This allows users to diagnose errors by inspecting Kubernetes resources with `kubectl`:
+
+```code
+$ kubectl get teleportusers myuser -o yaml
+```
+
+For example, if a user has been granted a nonexistent role the status will look like:
+
+```yaml
+apiVersion: resources.teleport.dev/v2
+kind: TeleportUser
+# [...]
+status:
+  conditions:
+  - lastTransitionTime: "2022-07-25T16:15:52Z"
+    message: Teleport resource has the Kubernetes origin label.
+    reason: OriginLabelMatching
+    status: "True"
+    type: TeleportResourceOwned
+  - lastTransitionTime: "2022-07-25T17:08:58Z"
+    message: 'Teleport returned the error: role my-non-existing-role is not found'
+    reason: TeleportError
+    status: "False"
+    type: SuccessfullyReconciled
+```
+
+Here `SuccessfullyReconciled` is `False` and the error is `role my-non-existing-role is not found`.
+
+If the status is not present or does not give sufficient information to solve the issue, check the operator logs:
+
+```shell
+$ kubectl logs "$PROXY_POD" -c operator
+```
+
+<Admonition type="note">
+  In case of multi-replica deployments, only one operator instance is running the reconciliation loop. This operator
+  is called the leader and is the only one producing reconciliation logs. The other operator instances are waiting with
+  the following log:
+
+  ```
+  leaderelection.go:248] attempting to acquire leader lease teleport/431e83f4.teleport.dev...
+  ```
+
+  To diagnose reconciliation issues, you will have to inspect all pods to find the one reconciling the resources.
+</Admonition>
```

Comment:
Nit: Add a "next steps" section at the end with a few links to e.g. Helm chart reference, access controls and [deployment guides](https://goteleport.com/docs/setup/helm-deployments/).

See other guides for examples, e.g. database access.

Reply:
Done